baseframe/templates/baseframe/bootstrap3/delete.html.jinja2

```diff
@@ -1,4 +1,4 @@
-{% extends "baseframe/bootstrap3/formlayout.html.jinja2" %}
+{% extends "formlayout.html" %}
```

Comment:
Any reason for reverting to `formlayout.html`?

Reply:
Since Hasjob has its own  formlayout.html